agent_build_refactored/managed_packages/steps/download_build_dependencies/download_build_dependencies.sh

```diff
@@ -18,29 +18,21 @@
 #   SOURCE_ROOT: Path to the projects root.
 #   STEP_OUTPUT_PATH: Path to the step's output directory.
 #
-# This script downloads source files for all libraries that are required to build Python. Since we build Python and its
-# requirements in a legacy OS (such as Centos 6) to achieve binary compatibility, we also have to build many essential
-# build tools because tools from legacy OS's ate too old.
+# This script downloads source files for all libraries that are required to build Python.
 #
 # It expects next environment variables:
 #
 #   XZ_VERSION: version of XZ Utils to build. Python requirement. ALso required by some make and configure scripts.
-#   PERL_VERSION: version of Perl to build. Required by some make and configure scripts.
```

Comment:
I found out that even centos:6 can provide more recent build tools, so we don't have to build them, only python dependencies.